components/Python/statistics/predictionStats/component.json

```diff
@@ -0,0 +1,48 @@
+{
+  "engineType": "Python",
+  "userStandalone": false,
+  "language": "Python",
+  "name": "predictionStats",
+  "label": "H2O predictionStats",
+  "description": "Read the H2O prediction file and generates statistics.",
+  "program": "stats.py",
+  "componentClass": "StatsReport",
+  "group": "Statistics",
+  "useMLOps": true,
+  "inputInfo": [{
+    "description": "File name",
+    "label": "File-Name",
+    "defaultComponent": "",
+    "type": "str",
+    "group": "data"
+    }
+  ],
+  "outputInfo": [],
+  "arguments": [
+    {
+      "key": "alert",
+      "label": "Alert Mechanishm",
+      "description": "Binary value indicating if the alert should be raised",
```

Comment:
Do we have boolean support?

Reply:
yes we do. I changed it to be 0/1 and verified that it works in both cases.